cluster-install/install-example-config.env

```diff
@@ -21,3 +21,6 @@ TRIDENT_ACP_IMAGE_TAG=24.02
 # Optional Connector configuration
 CONNECTOR_SKIP_TLS_VALIDATION=true
 
+# Autosupport
+CONNECTOR_AUTOSUPPORT_ENROLLED=false
+CONNECTOR_AUTOSUPPORT_URL=https://stagesupport.netapp.com/put/AsupPut
```

Comment:
Since this is a public repo, change enrolled to true and remove url altogether since customers should never need to change it.